commit 203e71625da03acc1f8670b2dd28cfa895a6bc88 (refs/changes/03/3403/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-16T14:45:27-08:00 (1 year, 8 months ago)
    
    TRITON-139 Triton's config-agents should poll at 120s interval as a stopgap

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 17fc5c8..bda632a 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 17fc5c8b83623078388b00260e5ad4be4f7348ac
+Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
